### PR TITLE
Improve compatibility with Jetbrains IDEs

### PR DESCRIPTION
**Pull request by ICYPhoenix7**

### CODE REVIEW

File: src/invoker/invoker.hpp
Comment (ICYPhoenix7) [inline src/invoker/invoker.hpp:1]: Didn't intend to commit this part, but for some reason that semicolon causes a bug in my syntax highlighting. Removing it did not cause any errors or warnings

**Comment by gir489returns** [comment]
Why?

**Comment by ICYPhoenix7** [comment]
> Why?

Screenshot

File: src/invoker/invoker.hpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ namespace big
 		constexpr void end_call()
 		{
 			// TODO: try to get rid of this
-			[[assume((cache_handlers(), m_are_handlers_cached == true))]];
+			[[assume((cache_handlers(), m_are_handlers_cached == true))]]
```
Comment (gir489returns) [inline src/invoker/invoker.hpp:40]: These semi-colons don't belong affixed to an attribute anyway. To the compiler, you basically just said:

"Hello, compiler. Please assume that Cache Handler's member are handlers cached is true. Thank you.
(New command)
End of Line."

 Good find.

**Review by gir489returns — APPROVED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**